lib/api/unittest/CDataFrameAnalysisRunnerTest.cc

```diff
@@ -95,12 +95,8 @@ BOOST_AUTO_TEST_CASE(testComputeAndSaveExecutionStrategyDiskUsageFlag) {
                         .diskUsageAllowed(false)
                         .outlierSpec();
 
-        // single error is registered that the memory limit is to low
-        LOG_DEBUG(<< "errors = " << core::CContainerPrinter::print(errors));
-        core::CRegex re;
-        re.init("Input error: memory limit.*");
-        BOOST_REQUIRE_EQUAL(1, static_cast<int>(errors.size()));
-        BOOST_TEST_REQUIRE(re.matches(errors[0]));
+        // no error should be registered
+        BOOST_REQUIRE_EQUAL(0, static_cast<int>(errors.size()));
```

Comment:
Casting to `int` is a leftover from CppUnit, which required the LHS and RHS types to be exactly the same.  Boost.Test doesn't, so the cast is just noise now.  But no need to make another commit just to change this, as it had the cast before too.